test/NuGet.Licenses.Tests/LicenseFileServiceFacts.cs

```diff
@@ -0,0 +1,187 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Moq;
+using System;
+using System.IO;
+using Xunit;
+
+namespace NuGet.Licenses.Tests
+{
+    public class LicenseFileServiceFacts
+    {
+        private static string TestLicenseIdentifier = "MIT";
+        private static string TestLicenseContent = "MIT license content";
+        private static string TestLicensesFolderPath = "TestFolder\\TestPath\\";
+
+        private Mock<ILicensesFolderPathService> _licensesFolderPathService;
+        private Mock<IFileService> _fileService;
+
+        public LicenseFileServiceFacts()
+        {
+            _licensesFolderPathService = new Mock<ILicensesFolderPathService>();
+            _licensesFolderPathService
+                .Setup(x => x.GetLicensesFolderPath())
+                .Returns(TestLicensesFolderPath);
+
+            _fileService = new Mock<IFileService>();
+            _fileService
+                    .Setup(x => x.GetFileFullPath(It.IsAny<string>()))
+                    .Returns(Path.Combine(TestLicensesFolderPath, TestLicenseIdentifier, ".txt"));
+            _fileService
+                .Setup(x => x.DoesFileExist(It.IsAny<string>()))
+                .Returns(true);
+            _fileService
+                .Setup(x => x.ReadFileContent(It.IsAny<string>()))
+                .Returns(TestLicenseContent);
+        }
+
+        private ILicenseFileService CreateService(
+            Mock<ILicensesFolderPathService> licensesFolderPathService = null,
+            Mock<IFileService> fileService = null)
+        {
+            if (licensesFolderPathService == null)
+            {
+                licensesFolderPathService = _licensesFolderPathService;
+            }
+
+            if (fileService == null)
+            {
+                fileService = _fileService;
+            }
+
+            return new LicenseFileService(licensesFolderPathService.Object, fileService.Object);
+        }
+
+        [Fact]
+        public void CheckLicenseFileExists()
+        {
+            // Arrange
+            var newService = CreateService();
+
+            // Act
+            var doesFileExist = newService.DoesLicenseFileExist(TestLicenseIdentifier);
+
+            // Assert
+            Assert.True(doesFileExist);
+            _licensesFolderPathService.Verify(
+                x => x.GetLicensesFolderPath(),
+                Times.Exactly(2));
+            _fileService.Verify(
+                x => x.DoesFileExist(It.IsAny<string>()),
```

Comment:
Here we probably want to check that the method is called with specific filename, not just with any.